modules/core/04-channel/v2/keeper/packet_test.go

```diff
@@ -120,28 +86,34 @@ func (suite *KeeperTestSuite) TestSendPacket() {
 			path = ibctesting.NewPath(suite.chainA, suite.chainB)
 			path.SetupV2()
 
+			packetData := mockv2.NewMockPacketData(mockv2.ModuleNameA, mockv2.ModuleNameB)
+
+			timeoutTimestamp := uint64(suite.chainB.GetContext().BlockTime().Add(time.Hour).Unix())
+
 			// create standard packet that can be malleated
-			packet = channeltypes.NewPacketWithVersion(mock.MockPacketData, 1, mock.PortID,
-				path.EndpointA.ChannelID, mock.PortID, path.EndpointB.ChannelID, clienttypes.NewHeight(1, 100), 0, mock.Version)
+			packet = types.NewPacket(1, path.EndpointA.ChannelID, path.EndpointB.ChannelID,
+				timeoutTimestamp, packetData)
 
 			// malleate the test case
 			tc.malleate()
 
 			// send packet
-			seq, err := suite.chainA.App.GetPacketServer().SendPacket(suite.chainA.GetContext(), packet.SourceChannel, packet.SourcePort,
-				packet.DestinationPort, packet.TimeoutHeight, packet.TimeoutTimestamp, packet.AppVersion, packet.Data)
+			seq, destChannel, err := suite.chainA.App.GetIBCKeeper().ChannelKeeperV2.SendPacketTest(suite.chainA.GetContext(), packet.SourceChannel, packet.TimeoutTimestamp, packet.Data)
 
 			expPass := tc.expError == nil
 			if expPass {
 				suite.Require().NoError(err)
+				// verify send packet method instantiated packet with correct sequence and destination channel
 				suite.Require().Equal(uint64(1), seq)
```

Comment:
worth adding an additional test case which ensures the sequence gets incremented on subsequent calls?